### PR TITLE
Milestone: Events Handler

### PR DESCRIPTION
**Pull request by Gleennkar**
## Add Events Handler
- Check for linter errors
- Updated my README.md file
- Add the logic directory which contains the calculate.js and operate.js
- Add event handler to give the calculator app its functionality

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,32 +1,48 @@
...
-        <button type="button" className="button">.</button>
-        <button type="button" className="symbol">=</button>
+        <button type="button" className="button" onClick={this.handleClick}>AC</button>
+        <button type="button" className="button" onClick={this.handleClick}>+/-</button>
+        <button type="button" className="button" onClick={this.handleClick}>%</button>
+        <button type="button" className="symbol active" onClick={this.handleClick}>÷</button>
+        <button type="button" className="button" onClick={this.handleClick}>7</button>
+        <button type="button" className="button" onClick={this.handleClick}>8</button>
+        <button type="button" className="button" onClick={this.handleClick}>9</button>
+        <button type="button" className="symbol active" onClick={this.handleClick}>&times;</button>
```
Comment (zainsadaqat) [inline src/components/Calculator.js]: The multiplication Operation isn't working. When I try to apply multiplication the calculator gets disappeared.


https://user-images.githubusercontent.com/83048208/198310756-39e0dae5-20f1-475c-bec3-e2ecceccaab2.mov

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Gleennkar  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Whoistolu) [reply src/components/Calculator.js]: Well resolved @Gleennkar 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Gleennkar 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**